docs/rules.md

```diff
@@ -1213,6 +1213,12 @@ Validates that system data is not defined in the properties bag, but rather as a
 
 Please refer to [system-data-in-properties-bag.md](./system-data-in-properties-bag.md) for details.
 
+### TagsAreNotAllowedForProxyResources
```

Comment:
Nit: consider renaming this TagsNotAllowedInPropertiesBag since it's not only for ProxyOnly resource types

Reply:
Initially named this "TagsAreTopLevelPropertiesOnly" due to suggestion changed to proxy
Will change to this "TagsNotAllowedInPropertiesBag"